components/ImprovedSearch/PaginationItem.tsx

```diff
@@ -0,0 +1,92 @@
+import { Button, List, ListItemProps } from "@equinor/eds-core-react"
+import { tokens } from "@equinor/eds-tokens"
+import { usePagination } from "react-instantsearch-hooks-web"
+import styled from "styled-components"
+
+const PaginationLink = styled(Button)<{ isCurrent?: boolean }>`
+  --button-size: 44px;
+  width: var(--button-size);
+  height: var(--button-size); 
+  color: tokens.colors.text.static_icons__default.hex} 
+  ${({ isCurrent }) =>
+    isCurrent && {
+      background: tokens.colors.interactive.primary__resting.hex,
+      color: tokens.colors.text.static_icons__default.hex,
+    }}
+  :hover {
+    color: tokens.colors.interactive.primary__resting.hex;
+
+    :disabled {
+      color: tokens.colors.ui.background__medium.hex;
+    }
+  }
+
+  :disabled {
+    cursor: auto;
+    color: tokens.colors.ui.background__medium.hex;
+  }
+`
+const { Item: EdsItem } = List
+
+const StyledListItem = styled(EdsItem)``
+
+export const isModifierClick = (event: React.MouseEvent) => {
+  const isMiddleClick = event.button === 1
+  return Boolean(isMiddleClick || event.altKey || event.ctrlKey || event.metaKey || event.shiftKey)
```

Comment:
this disables default operating system behaviors. for instance, a middle mouse click on a link on windows, will open the link in a new tab. likewise with a cmd + left click on mac. i don't think it's a good idea to block operating system behaviors